### PR TITLE
Adding an Easyconfig for Minimac2

### PR DESCRIPTION
**Pull request by rtripath89**

### CODE REVIEW

File: easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
```
Comment (boegel) [inline easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb]: @rtripath89 wouldn't it be better to recompile rather using the shipped binaries? similar to `Minimac`, see https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/m/Minimac/Minimac-20140110-goolf-1.4.10.eb ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2066.delcatty.os - Linux SL 6.8, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/aa12db0c1b4d26c765b98a8c16d00bbe for a full test report.

File: easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb]: @pescobar thoughts on this?

Comment (pescobar) [inline easybuild/easyconfigs/m/Minimac2/Minimac2-2014.9.15-intel-2015b.eb]: @boegel I also prefer to build from sources when possible. I have written a easyconfig which builds from sources

https://github.com/hpcugent/easybuild-easyconfigs/pull/3783

**Comment by boegel** [comment]
@rtripath89 do you mind updating this as is done in #3783?

**Comment by migueldiascosta** [comment]
@rtripath89, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**